twine-upload.sh

```diff
@@ -45,7 +45,7 @@ if [[ ${INPUT_VERBOSE,,} != "false" ]] ; then
 fi
 
 if [[ ${INPUT_PRINT_HASH,,} != "false" || ${INPUT_VERBOSE,,} != "false" ]] ; then
-    python /app/print-hash.py "${INPUT_PACKAGES_DIR%%/}"
+    python /app/print-hash.py ${INPUT_PACKAGES_DIR%%/}
```

Comment:
I feel like this patch may end up opening a can of worms because unquoted variable interpolation may turn into multiple arguments if it has spaces in it.
Have you checked such problematic scenarios? I'd very much like to be more confident with this...
Also, why not change this script to work the same way as other commands accepting `${INPUT_PACKAGES_DIR%%/}/*`?
Any ideas?

Reply:
I just had a few tests on it.

For example, a path like `/github/workspace/dist package/` may fail all twine commands since there is a space in the path and it is not quoted. 
Also, error like this will be displayed:
```
Warning:  It looks like there are no Python distribution packages to publish in the directory 'dist package/'. Please verify that they are in place should you face this problem.
```
Because these arguments are not quoted:
https://github.com/pypa/gh-action-pypi-publish/blob/1bbe3c9926702534a7e250bcb3c6ea3cd787b53b/twine-upload.sh#L21-L22

https://github.com/pypa/gh-action-pypi-publish/blob/1bbe3c9926702534a7e250bcb3c6ea3cd787b53b/twine-upload.sh#L35

https://github.com/pypa/gh-action-pypi-publish/blob/1bbe3c9926702534a7e250bcb3c6ea3cd787b53b/twine-upload.sh#L54

Reply:
> Also, why not change this script to work the same way as other commands accepting `${INPUT_PACKAGES_DIR%%/}/*`?

Because `iterdir()` is used so using `${INPUT_PACKAGES_DIR%%/}/*` will result in NotADirectoryError.

Reply:
I meant that you wouldn't need to use `iterdir()` because the outer shell would resolve that into multiple arguments IIUC.